src/lib/EventSubscriber/AppRendererSubscriber.php

```diff
@@ -51,7 +51,7 @@ public function renderApp(FilterResponseEvent $event)
 
         $response = $event->getResponse();
 
-        if ($response->isRedirect()) {
+        if ($response->isRedirect() || $response instanceof NoRenderResponse) {
```

Comment:
there will be other Response classes that use this, isRedirect is in the parent and checks for the 3xx codes, I don't like to override it and class something that isn't technically a redirect as a redirect.